hazelcast/src/main/java/com/hazelcast/query/Predicates.java

```diff
@@ -29,77 +29,342 @@
 import com.hazelcast.query.impl.predicates.OrPredicate;
 import com.hazelcast.query.impl.predicates.RegexPredicate;
 
+import java.util.Date;
+
 /**
  * A utility class to create {@link com.hazelcast.query.Predicate} instances.
+ * <p>
+ * <b>Special Attributes</b>
+ * <p>
+ * The predicate creation methods accepting an attribute name support two special attributes:
```

Comment:
"creation methods"
It's not wrong, but "factory method" is more common.

Reply:
fixed